src/components/LiquidityMining/ConfirmStakingRewardsDistributionCreation/StakingRewardsDistributionCreationModalFooter.tsx

```diff
@@ -67,7 +74,7 @@ export default function StakingRewardsDistributionCreationModalFooter({
             paddingLeft: '10px',
           }}
         >
-          {reward ? `${reward.toExact()} ${reward.token.symbol}` : '-'}
+          {reward ? reward.map(ite => `${ite.toExact()} ${ite.token.symbol}`).join(', ') : '-'}
```

Comment:
I think this naming is better, its self-explanatory.

```suggestion
          {rewards ? rewards.map(reward => `${reward.toExact()} ${reward.token.symbol}`).join(', ') : '-'}
```